### PR TITLE
fix: Enable ADC resolution

### PR DESCRIPTION
**Pull request by sethvargo**
Closes https://github.com/google-github-actions/get-gke-credentials/pull/124

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -31,20 +39,39 @@ async function run(): Promise<void> {
     const name = getInput('cluster_name', { required: true });
     const location = getInput('location', { required: true });
```
Comment (bharathkkb) [inline src/main.ts:40]: Should we drop this as required since we now support grabbing it from resource name via `name` input?

**Review by bharathkkb — COMMENTED** [review]
LGTM, just a question

File: src/main.ts (hunk 1 above)
Comment (sethvargo) [reply src/main.ts]: Done

**Comment by sethvargo** [comment]
@bharathkkb PTAnotherL

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**